.github/workflows/test-sui.yaml

```diff
@@ -196,7 +196,7 @@ jobs:
           echo "emptyTokenName=Empty" >> $GITHUB_ENV
           echo "emptyTokenDecimals=6" >> $GITHUB_ENV
```

Comment:
set the hub chain and address as an env var to avoid duplication and for consistency